### ISSUE TITLE
Show extra dependencies from `Config/Needs/*` fields

### ISSUE THREAD

**Issue by eitsupi** [r-universe-org/help#230]
R-universe has the ability to display reverse dependent packages, but I was wondering if it is possible to display information on other fields `Config/Needs/*` (e.g. `Config/Needs/website`) here. (Related to r-lib/remotes#459)

This is a feature that CRAN does not have, but it would be great to be able to notify people about packages they need, for example during development.

**Comment by eitsupi** [comment]
Sorry, I should have posted this in the discussion...... (I realized too late that discussions are enabled in this repository)

**Comment by jeroen** [comment]
Yes we could do that. The information is already in the `/json` api e.g: https://r-lib.r-universe.dev/pkgdown/json. We could display it on the front-end.
